web/src/main/java/it/pagopa/selfcare/dashboard/web/config/DashboardSecurityConfig.java

```diff
@@ -0,0 +1,48 @@
+package it.pagopa.selfcare.dashboard.web.config;
+
+import it.pagopa.selfcare.commons.web.config.SecurityConfig;
+import it.pagopa.selfcare.commons.web.security.JwtService;
+import it.pagopa.selfcare.dashboard.connector.rest.client.PartyProcessRestClient;
+import it.pagopa.selfcare.dashboard.web.security.PartyAuthenticationProvider;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.TestingAuthenticationProvider;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.core.authority.mapping.SimpleAuthorityMapper;
+
+@Slf4j
+@Configuration
+class DashboardSecurityConfig extends SecurityConfig {
+
+    private final PartyProcessRestClient restClient;
+
+
+    @Autowired
+    public DashboardSecurityConfig(JwtService jwtService, PartyProcessRestClient restClient) {
+        super(jwtService);
+        this.restClient = restClient;
+    }
+
+
+    @Override
+    public void configure(AuthenticationManagerBuilder authenticationManagerBuilder) {
```

Comment:
Should this configuration be implemented in SecurityConfig?

Reply:
we will move this logic into the joint project after [SELC-180](https://pagopa.atlassian.net/browse/SELC-180)